.github/workflows/nightly-rosetta-t5x-build-test.yaml

```diff
@@ -31,79 +31,117 @@ permissions:
-          echo "PUBLISH=${{ inputs.PUBLISH }}" >> $GITHUB_OUTPUT
-
-  build:
-    if: (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') || github.event_name == 'workflow_dispatch'
+          
+  amd64:
     needs: metadata
     uses: ./.github/workflows/_build_rosetta.yaml
     with:
+      ARCHITECTURE: amd64
       BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
       BASE_LIBRARY: ${{ needs.metadata.outputs.BASE_LIBRARY }}
       BASE_IMAGE: ${{ needs.metadata.outputs.BASE_IMAGE }}
-      # TODO: Can't build ARM until https://github.com/NVIDIA/JAX-Toolbox/pull/252 is available
-      PLATFORMS: '["amd64"]'
     secrets: inherit
+      
+  arm64:
+    needs: metadata
+    runs-on: ubuntu-22.04
+    outputs:
+      DOCKER_TAG_MEALKIT: ''
+    steps:
+      - name: Generate placeholder warning
+        shell: bash -x -e {0}
+        run: |
+          echo "WARNING: arm64 build is not yet supported"
+    
+  publish-mealkit:
+    needs: [metadata, amd64, arm64]
+    if: needs.metadata.output.PUBLISH == 'true' 
+    uses: ./.github/workflows/_publish_container.yaml
+    with:
+      SOURCE_IMAGE: |
+        ${{ needs.amd64.outputs.DOCKER_TAG_MEALKIT }}
+        ${{ needs.arm64.outputs.DOCKER_TAG_MEALKIT }}
+      TARGET_IMAGE: upstream-t5x
+      TARGET_TAGS: |
+        type=raw,value=mealkit,priority=500
+        type=raw,value=mealkit-${{ needs.metadata.outputs.BUILD_DATE }},priority=500  
 
-  publish-build:
-    needs: [metadata, build]
-    uses: ./.github/workflows/_publish_badge.yaml
-    if: ( always() )
-    secrets: inherit
+  publish-final:
+    needs: [metadata, amd64, arm64]
+    if: needs.metadata.outputs.PUBLISH == 'true'
+    uses: ./.github/workflows/_publish_container.yaml
     with:
-      ENDPOINT_FILENAME: 'rosetta-t5x-build-status.json'
-      PUBLISH: ${{ github.event_name == 'workflow_run' || needs.metadata.outputs.PUBLISH == 'true' }}
-      SCRIPT: |
-        if [[ ${{ needs.build.result }} == "success" ]]; then
-          BADGE_COLOR=brightgreen
-          MSG=passing
-        else
-          BADGE_COLOR=red
-          MSG=failing
-        fi
-        echo "LABEL='nightly'" >> $GITHUB_OUTPUT
-        echo "MESSAGE='${MSG}'" >> $GITHUB_OUTPUT
-        echo "COLOR='${BADGE_COLOR}'" >> $GITHUB_OUTPUT
+      SOURCE_IMAGE: |
+        ${{ needs.amd64.outputs.DOCKER_TAG_FINAL }}
+        ${{ needs.arm64.outputs.DOCKER_TAG_FINAL }}
+      TARGET_IMAGE: upstream-t5x
+      TARGET_TAGS: |
+        type=raw,value=latest,priority=1000
+        type=raw,value=nightly-${{ needs.metadata.outputs.BUILD_DATE }},priority=900  
 
   test-unit:
     if: (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') || github.event_name == 'workflow_dispatch'
-    needs: build
+    needs: [metadata, amd64, arm64]
     uses: ./.github/workflows/_test_rosetta.yaml
     with:
       ROSETTA_IMAGE: ${{ needs.build.outputs.DOCKER_TAGS }}
```

Comment:
I think `ROSETTA_IMAGE` needs to be updated here

Reply:
Done